content/installation/python/PythonOverview.md

```diff
@@ -4,16 +4,24 @@ description: "Troubleshooting the Python Agent"
 tags: "installation python agent troubleshooting"
 -->
 
-The Contrast Python agent provides runtime protection of Django, Flask and Pyramid web applications. 
+The Contrast Python agent provides runtime protection of Django, Flask and Pyramid web applications.
+
+Version 2.3.0 of the agent introduces beta Assess functionality. This provides the ability to find vulnerabilities in applications before they can be exploited.
+
+>**Note:** The beta Assess feature can only be licensed in specific instances of the Contrast UI.
 
```

Comment:
@drdavella What does "specific instances" mean here?

Reply:
The beta version only works in `alpha` and `eval` right now, so it won't be available for all users. There's probably a better way to say that, though.